---
map_frame: "base_link"
map_res: 0.4
map_x: -20.2
map_y: -20.2
map_z: 0
map_height: 100
map_width: 100
use_hist: true
hist_dist: 2
anchor_frame: "odom"
